scripts/deployment/phase2-assets/collaterals/deploy_bbusdt.ts

```diff
@@ -0,0 +1,91 @@
+import fs from 'fs'
+import hre from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { networkConfig } from '../../../../common/configuration'
+import { fp } from '../../../../common/numbers'
+import { expect } from 'chai'
+import { CollateralStatus } from '../../../../common/constants'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import {
+  USDT_ORACLE_TIMEOUT,
+  USDT_ORACLE_ERROR,
+  USDT_USD_FEED,
+  PRICE_TIMEOUT,
+  DELAY_UNTIL_DEFAULT,
+} from '../../../../test/plugins/individual-collateral/meta-morpho/constants'
+import { MetaMorphoFiatCollateral } from '../../../../typechain'
+import { ContractFactory } from 'ethers'
+
+async function main() {
+  // ==== Read Configuration ====
+  const [deployer] = await hre.ethers.getSigners()
+
+  const chainId = await getChainId(hre)
+
+  console.log(`Deploying Collateral to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+  if (!networkConfig[chainId]) {
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+
+  /********  Deploy MetaMorpho Flagship USDT - bbUSDT  **************************/
+
+  const MetaMorphoFiatCollateralFactory: ContractFactory = await hre.ethers.getContractFactory(
+    'MetaMorphoFiatCollateral'
+  )
+
+  const collateral = <MetaMorphoFiatCollateral>await MetaMorphoFiatCollateralFactory.connect(
+    deployer
+  ).deploy(
+    {
+      priceTimeout: PRICE_TIMEOUT.toString(),
+      chainlinkFeed: USDT_USD_FEED,
+      oracleError: USDT_ORACLE_ERROR.toString(),
+      erc20: networkConfig[chainId].tokens.bbUSDT,
+      maxTradeVolume: fp('1e6').toString(), // $7.5m vault
```

Comment:
I think we should be using _much_ smaller numbers for this. The vaults are currently too small to sustain these.

Reply:
Ok I think $500k can make sense, but much lower and things become unusable on mainnet

Reply:
And <$500k for pyUSD, since it has $1.7m vault

Reply:
That sounds good to me. Probably $250k for smaller vaults and $500k for every other.